views/package.html.twig

```diff
@@ -96,13 +96,12 @@
                 {%- if package.highest.type == 'metapackage' -%}
                 {{ version.prettyVersion }}
```

Comment:
this should also use pills, otherwise the display is broken for meta packages due to removing the comma.